library/general/src/modules/Popup.rb

```diff
@@ -1029,6 +1033,21 @@ def Feedback(headline, message, &block)
       ClearFeedback()
     end
 
+    # Hides feedback during block execution.
+    # After block it is shown again. When exception is raised, then it is
+    # shown again.
```

Comment:
I do not see any `ensure`, how is that open again?

Reply:
Am typo in documentation. I want to have there "it is NOT shown again". Reason is that during exception it probably want to show something different then feedback that issue happen.

Reply:
Ah, then OK. :smiley: